services/app/src/components/loginPage/LoginForm.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useContext, useState } from 'react';
+import { useNavigate } from 'react-router';
+import { WsContext } from '../../contexts/ws-context';
+import './loginPage.css'
+import userInputIcon from './SVGs/userInputIcon.svg';
+
+const LoginForm = () => {
+  const navigate = useNavigate();
+  const navigateUrl = '/game/test';
+  const { ws } = useContext(WsContext);
+
+  const [btnIsDisabled, setBtnIsDisabled] = useState(true);
+
+  const submitInfo = (event) => {
+    event.preventDefault();
+    window.localStorage.setItem('user', event.target.username.value);
+    navigate(navigateUrl);
+    ws.send(
+      JSON.stringify({
```

Comment:
Have a look at this file `services/app/src/api/playerApi.js` it contains action builders for WS events, extend that file with this event.